global-api/routes/ccra_assessment.py

```diff
@@ -0,0 +1,212 @@
+              		a.sensitivity_score,
+              		a.risk_lower_limit,
+              		a.risk_upper_limit
+            FROM modelled.ccra_riskassessment a
+            LEFT JOIN modelled.ccra_impactchain b
+            ON a.impact_id = b.id
+            WHERE actor_id = :actor_id
+            AND b.scenario_name = :scenario_name
+            """
+        )
+
+        params = {
+            "actor_id": actor_id,
+            "scenario_name": scenario_name,
+        }
+        result = session.execute(query, params).mappings().all()
+
+    return result
+
+def db_impactchain_indicator(actor_id, scenario_name):
+    with SessionLocal() as session:
+        query = text(
+            """
+            SELECT 	c.keyimpact_name,
+              		c.hazard_name,
+              		c.scenario_name,
+                    a.actor_id,
+              		a.category,
+              		a.subcategory,
+              		a.indicator_name,
+              		b.indicator_score,
+              		b.indicator_units,
+              		b.indicator_normalized_score,
+              		a.indicator_weight,
+              		a.relationship,
+              		b.indicator_year,
+              		b.datasource
+            FROM modelled.ccra_impactchain_indicator a
+            LEFT JOIN modelled.ccra_indicator b
+            ON a.indicator_id = b.id
+            LEFT JOIN modelled.ccra_impactchain c
+            ON a.impact_id = c.id
+            WHERE a.actor_id = :actor_id
+            AND c.scenario_name = :scenario_name
+            """
+        )
+
+        params = {
+            "actor_id": actor_id,
+            "scenario_name": scenario_name,
+        }
+        result = session.execute(query, params).mappings().all()
+
+    return result
+
+
+def db_ccra_cities(country_code):
+    with SessionLocal() as session:
+        query = text(
+            """
+            SELECT 	city_name,
+              		region_code,
+              		locode as actor_id,
+              		'R' || osm_id as osm_id
+            FROM 	modelled.city_polygon
+            WHERE 	country_code = :country_code
+            """
+        )
+
+        params = {
+            "country_code": country_code,
+        }
+        result = session.execute(query, params).mappings().all()
+
+    return result
+
+
+@api_router.get("/ccra/city/{country_code}", summary="Get ccra cities")
+def get_ccra_cities(
+    country_code: str,  # This is the two letter country code (ISO 3166-1 alpha-2)
+    ):
+    """
+    Retrieve ccra risk assessment based on specified parameters.
+
+    - `country_code`: Unique identifier for the country, e.g BR for Brazil
+
+    """
+
+    records = db_ccra_cities(country_code)
+
+    if not records:
+        raise HTTPException(status_code=404, detail="No data available")
+
+    # Transform database records into the desired format for the response
+    response = [
+            {
+                "cityName": record['city_name'],  # Assuming actor_id corresponds to city name
```

Comment:
Can we use consistently either camelCase or snake_case for the keys and not mix it?